common/src/thrift/tachyon.thrift

```diff
@@ -220,6 +220,9 @@ service FileSystemMasterService {
   void setPinned(1: i64 fileId, 2: bool pinned)
     throws (1: TachyonTException e)
 
+  void setTTL(1: i64 fileId, 2: i64 ttl)
```

Comment:
Instead of doing this, why not have a single RPC called SetState, that takes SetStateTOptions? That way, we don't have to modify the RPC interface every time we want to add support for mutating another aspect of the file state.